java/jdbc/src/test/java/io/vitess/jdbc/VitessResultSetTest.java

```diff
@@ -495,6 +497,17 @@ public Cursor getCursorWithRowsAsNull() {
         // .getAsciiStream(19));
     }
 
+    @Test public void testGetBinaryStream() throws SQLException, IOException {
+        Cursor cursor = getCursorWithRows();
+        VitessResultSet vitessResultSet = new VitessResultSet(cursor, getVitessStatement());
+        vitessResultSet.next();
+        byte[] ba1 = new byte[128];
+        new ByteArrayInputStream("HELLO TDS TEAM".getBytes()).read(ba1, 0, 128);
```

Comment:
According to code coverage, this test does not cover the switch statement at all -- just the fallthrough `new ByteArrayInputStream` case. This is because col19 is TEXT and non-binary, so gets set to VARCHAR (in FieldWithMetadata.java). VARCHAR gets handled with getBytes(). Can you add assertions to cover the switch statement cases and the null fallthrough?